OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -51,8 +51,20 @@ public void RenderAfterWorld(WorldRenderer wr, Actor self)
 			var iz = 1 / wr.Viewport.Zoom;
 
 			if (healthInfo != null)
+			{
 				healthInfo.Shape.DrawCombatOverlay(wr, wcr, self);
 
+				var targetZc = Color.Magenta;
+				var tx = new WVec(96, 0, 0);
```

Comment:
Okay, these look to be width and height values.

>Yes. But nearly all other debug overlays we have aren't customizable, either.

I find that to be a poor reason not to allow someone to customize something, but ok for consistency.